### ISSUE TITLE
RIKEN: Connect Equipment and test communication

### ISSUE THREAD

**Issue by davidkeymer** [ISISComputingGroup/IBEX#7339]
The new PSUs (33x Danfysik, 6x Transtechnik, 3x Technix), original RB2 (2x 'Danfysik') and pressure controllers (~11x TPG300) need to be connected to the new MOXA NPort devices.

## Acceptance criteria 
- [ ] Equipment connected
- [ ] IOCs configured and communication tested

## Notes

- Flying leads from each piece of equipment terminate at each MOXA position.  Care needs to be taken connecting these to an appropriate socket and configuring the IOC accordingly.
- Ideally #7145 should be completed first to save duplication of work

**Comment by ChrisM-S** [comment]
Do we need a separate ticket to establish communication with RB2 and to test as it does involve re-wiring and testing connections (perhaps with a laptop initially)

**Comment by davidkeymer** [comment]
All equipment mentioned in description above (including the Schneider PLC discussed in #7241) has now been connected and communications successfully tested.  IOCs configured accordingly in newly created components.

RB2 is currently using its original PSU, which has also been successfully tested and appropriate modifications to the Danfysik driver made.  PRs pending.

Documentation still needs to be compiled from various sources and summarized on the wiki page.

**NB** Connections to MOXA ports were made by the Electrical Support group.  Apart from one mix-up, all were correct and as expected.

**Comment by rerpha** [comment]
electrical support group have seen it running, PSUs now off
